versions.props

```diff
@@ -70,3 +70,4 @@ org.semver4j:semver4j=5.2.1
 org.slf4j:*=2.0.9
 org.xerial.snappy:snappy-java=1.1.10.5
 software.amazon.awssdk:*=2.20.155
+org.apache.opennlp:opennlp*=2.2.0
```

Comment:
Should be in order alphabetically...

Reply:
I think updating the Lucene dependency is a PR on you @jzonthemtn  right?  (I actually have done no commits to Lucene, so wouldn't be comfortable attempting to update opennlp there).   And yeah, then in Lucene 9.9, we could remove this dependency here...